x/shared/session_test.go

```diff
@@ -168,9 +168,9 @@ func TestClaimProofWindows(t *testing.T) {
 			sharedParams: sharedtypes.Params{
```

Comment:
These sorts of bugs are where putting together a real unit test goes a really long way.

Reply:
This bug could not be observed by a unit test as its causes are mostly timing and latency.